PY_Homework001/taskHW06.py

```diff
@@ -0,0 +1,25 @@
+# Домашнее задание№6/Семинар№1:
+# ========================================================================================================
+# Вы пользуетесь общественным транспортом? Вероятно, вы расплачивались за проезд и получали билет с номером. Счастливым
+# билетом называют такой билет с шестизначным номером, где сумма первых трех цифр равна сумме последних трех. Т.е. билет с номером
+# 385916 – счастливый, т.к. 3+8+5=9+1+6. Вам требуется написать программу, которая проверяет счастливость билета.
+# =========================================================================================================
+# Решил: Артем Анатольевич Авдеев
+# ________________________________________________________________________________________________________
+
+string_digit = input("Введите шестизначное число: ")
+if len(string_digit) != 6:
+    print("Вы ввели не шестизначное число! Повторите ввод!")
+else:
+    digit_sum1 = 0
+    digit_sum2 = 0
+    for i in range(3):
+        digit_sum1 += int(string_digit[i])
+        digit_sum2 += int(string_digit[i + 3])
```

Comment:
Или так. Универсальный вариант
```suggestion
        digit_sum2 += int(string_digit[-i - 1])
```

Reply:
Спасибо! Учту!